backend/pkg/api/connect/service/transform/util.go

```diff
@@ -0,0 +1,79 @@
+// Copyright 2023 Redpanda Data, Inc.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.md
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0
+
+package transform
+
+import (
+	"fmt"
+	"strings"
+
+	"connectrpc.com/connect"
+	"github.com/redpanda-data/redpanda/src/go/rpk/pkg/adminapi"
+
+	apierrors "github.com/redpanda-data/console/backend/pkg/api/connect/errors"
+	commonv1alpha1 "github.com/redpanda-data/console/backend/pkg/protogen/redpanda/api/common/v1alpha1"
+	v1alpha1 "github.com/redpanda-data/console/backend/pkg/protogen/redpanda/api/dataplane/v1alpha1"
+)
+
+func findTransformByName(transforms []*v1alpha1.TransformMetadata, name string) (*v1alpha1.TransformMetadata, error) {
+	for _, transform := range transforms {
+		if strings.Contains(transform.Name, name) {
```

Comment:
Do you **really** want `Contains` here?

That means that `rpk transform delete foo` could delete a transform called `foobar`. Same with `get` requests. Maybe we should do exact equality?

Reply:
I followed topic on this one https://github.com/redpanda-data/console/blob/e724c8d3461e80aff70808fa710283a3f9171fbf/backend/pkg/api/connect/service/topic/service.go#L59

Reply:
But it does not use that for get and delete. I think it's fine for list